specification/workloads/resource-manager/Microsoft.Workloads/connectors/preview/2023-10-01-preview/connectors.json

```diff
@@ -0,0 +1,1098 @@
+      "x-ms-discriminator-value": "HANA",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupData"
+        }
+      ],
+      "required": [
+        "backupPolicy",
+        "hdbuserstoreKeyName"
+      ],
+      "properties": {
+        "sslConfiguration": {
+          "type": "object",
+          "$ref": "#/definitions/SSLConfiguration",
+          "description": "Path of the SSL key store."
+        },
+        "backupPolicy": {
+          "$ref": "#/definitions/DBBackupPolicyProperties"
+        },
+        "hdbuserstoreKeyName": {
+          "type": "string",
+          "description": "Name of the HANA Database User Store Key."
+        },
+        "instanceNumber": {
+          "description": "Gets or sets the database instance number.",
+          "type": "string"
+        },
+        "dbInstanceSnapshotBackupPolicy": {
+          "$ref": "#/definitions/DBBackupPolicyProperties"
+        }
+      }
+    },
+    "SqlBackupData": {
+      "description": "Defines the SQL Backup data for a virtual instance for SAP.",
+      "x-ms-discriminator-value": "SQL",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BackupData"
+        }
+      ],
+      "required": [
+        "backupPolicy"
+      ],
+      "properties": {
+        "backupPolicy": {
+          "$ref": "#/definitions/DBBackupPolicyProperties"
+        }
+      }
+    },
+    "ACSSBackupConnectionProperties": {
+      "description": "Defines the backup connection properties for a virtual instance for SAP.",
+      "type": "object",
+      "properties": {
+        "backupData": {
+          "$ref": "#/definitions/BackupData"
+        },
+        "provisioningState": {
+          "$ref": "#/definitions/ConnectorProvisioningState",
+          "readOnly": true
+        },
+        "errors": {
+          "description": "Defines the errors related to backup connection resource of virtual instance for SAP.",
+          "readOnly": true,
+          "$ref": "#/definitions/ConnectorErrorDefinition"
+        }
+      }
+    },
+    "BackupData": {
+      "description": "Information about the recovery services vault and backup policy used for backup.",
+      "type": "object",
+      "required": [
+        "backupType",
+        "recoveryServicesVault"
+      ],
+      "properties": {
+        "backupType": {
+          "$ref": "#/definitions/BackupType"
+        },
+        "recoveryServicesVault": {
+          "$ref": "#/definitions/RecoveryServicesVaultProperties"
+        }
+      },
+      "discriminator": "backupType"
+    },
+    "BackupType": {
+      "description": "The type of backup, VM, SQL or HANA.",
+      "type": "string",
+      "enum": [
+        "VM",
+        "SQL",
+        "HANA"
+      ],
+      "x-ms-mutability": [
+        "create",
+        "read"
+      ],
+      "x-ms-enum": {
+        "name": "backupType",
```

Comment:
>backupType [](http://example.com/codeflow?start=17&length=10)

I wonder if this should be BackupType.

Reply:
Since we have followed this naming convention of properties names starting with lower case in all the definitions, keeping it same here as well